src/main/java/AlexLink/HomeWork/HW7/City/City.java

```diff
@@ -0,0 +1,79 @@
+package AlexLink.HomeWork.HW7.City;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class City {
+    private String name;
+    private List<Title> listStreetsAndSquares;
```

Comment:
You use some strange value with absctract not inner class and inner class, which extends by this absctract. Usually, if you need inner class, it is created in out class like inner and doesn't extends another